src/tests/Microsoft.PowerFx.Interpreter.Tests/MutationScripts/Collect.txt

```diff
@@ -36,3 +36,9 @@ Table({Value:1},{Value:2},{Value:3},{Value:99},{Value:88},{Value:77},{Value:77},
 
 >> Collect(Table({x:1,y:2}), {x: 3}).y 
 Blank()
+
+>> Collect([1,2,3],{Value:"200"}).Value
```

Comment:
> Collect([1,2,3],{Value:"200"}).Value [](http://example.com/codeflow?start=2&length=37)

Can we do,
With( { t: [1,2,3] }, Collect( t, {Value:"200"}); t)
Just so we can see what mutated table looks like now.

Reply:
Added.